exporters/user_events/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ endif()
 add_definitions(-DHAVE_CONSOLE_LOG -DENABLE_LOGS_PREVIEW)
 
 if(MAIN_PROJECT)
+  find_package(protobuf REQUIRED)
```

Comment:
why is protobuf required ?

Reply:
Add it to build the main repo with OTLP support for later use. I'll clean this up.